views/index.ejs

```diff
@@ -1,25 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" type="text/css" href="../styles/reset.css" />
-  <link rel="stylesheet" type="text/css" href="../styles/base.css" />
-  <link rel="stylesheet" type="text/css" href="../styles/icons.css" />
-  <link rel="stylesheet" type="text/css" href="../styles/layout.css" />
-  <link rel="stylesheet" type="text/css" href="../styles/modules.css" />
-  <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
-  <title>Book App</title>
-</head>
+<%- include('partials/head'); %>
 <body>
-  <header>
-    <h1><%= bookTitle %></h1>
-  </header>
+  <%- include('partials/header', {bookTitle: 'Bookster'}); %>
+  
   <div class="container">
     <% books.forEach( book => {  %>
       <div class="galleryItem">
-        <a href="#"><img src="<%= book.image_url %>" alt=""/></a>          
+        <a href="/book/<%=book.id%>"><img src="<%= book.image_url %>" alt=""/></a>          
         <h4>Author: <%= book.author %></h4>
         <h3>Title: <%= book.title %></h3>
       </div>
```

Comment:
you have a footer.ejs but you're also using a <footer> in your index file. You should probably change the <footer> in index.ejs to just a <p> or something